tools/docker/bin/run.sh

```diff
@@ -94,7 +94,7 @@ for DIR in /usr/local/src/jetpack-monorepo/projects/plugins/*; do
 done
 
 # Symlink jetpack into wordpress-develop for WP >= 5.6-beta1
-if [ ! -e /tmp/wordpress-develop/tests/phpunit/data/plugins/jetpack ]; then
+if [ ! -L /tmp/wordpress-develop/tests/phpunit/data/plugins/jetpack ]; then
```

Comment:
Hm, I wonder how this situation may occur. Definitely, one can manually copy the folder, but why someone would want to do that?

Reply:
It shouldn't occur, unless someone messes around manually. If someone did do something like that, the first time it would create a symlink at /tmp/wordpress-develop/tests/phpunit/data/plugins/jetpack/jetpack and subsequent times it would give an error along the lines of "ln: failed to create symbolic link '/tmp/wordpress-develop/tests/phpunit/data/plugins/jetpack/jetpack': File exists".

Reply:
done in 
24ea224